code/modules/asset_cache/asset_cache_client.dm

```diff
@@ -0,0 +1,58 @@
+
+/client
+	var/list/sent_assets = list() // List of all asset filenames sent to this client by the asset cache, along with their assoicated md5s
+	var/list/completed_asset_jobs = list() /// List of all completed blocking send jobs awaiting acknowledgement by send_asset
+
+	var/last_asset_job = 0 /// Last asset send job id.
+	var/last_completed_asset_job = 0
```

Comment:
Move these variables to the client_defines file.

Reply:
but this is cache specific

Reply:
Thing is this sort of things fucks up the DM langserv for VSC from what I heard.

Reply:
only if you're specifically finding `/client`, but the vars is fine.